wscript

```diff
@@ -211,10 +211,17 @@ def configure(conf):
     if conf.env['IS_WINDOWS']:
         conf.env.append_unique('CCDEFINES', '_POSIX')
         conf.env.append_unique('CXXDEFINES', '_POSIX')
-
-    conf.env.append_unique('CXXFLAGS', '-Wall')
-    conf.env.append_unique('CXXFLAGS', '-std=gnu++11')
-    conf.env.append_unique('CFLAGS', '-Wall')
+        conf.env.append_unique('CXXFLAGS', '/std:c++14')
```

Comment:
Seems like it won't compile with mingw with these changes...

Reply:
Just by looking at your git profile, I assume you're involved with Radium? Maybe you have mingw compiler to test with? I wonder now if waf automatically chooses either mingw or msvc, if so, both need to work, I guess? Tough I don't see any trade-off for sticking with mingw when msvc works.